### PR TITLE
Add a smoke test for perf config.

### PR DESCRIPTION
**Pull request by paulirish**
Using a superfast local site, so we expect 100s on the scores.

Anything else I should add to the expectations?

### CODE REVIEW

File: lighthouse-cli/test/smokehouse/perf/expectations.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+/**
+ * Expected Lighthouse audit values for byte efficiency tests
```
Comment (patrickhulce) [inline lighthouse-cli/test/smokehouse/perf/expectations.js]: nit: remove

File: lighthouse-cli/test/smokehouse/perf/expectations.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+            timings: {
+            }
+          }
+        }
+      },
+      'time-to-interactive': {
+        score: 100,
+        extendedInfo: {
+          value: {
+            foundLatencies: {
```
Comment (patrickhulce) [inline lighthouse-cli/test/smokehouse/perf/expectations.js]: hang on I moved this to `latencies.timeToInteractive` in my PR and I just pushed it but you already merged 😬 

do people rely on this?

Comment (paulirish) [inline lighthouse-cli/test/smokehouse/perf/expectations.js]: nope. they dont.

**Comment by paulirish** [comment]
ptal

**Pull request merged**